test/qunit/model/FHIRModel.integration.js

```diff
@@ -660,4 +663,31 @@ sap.ui.define([
 		this.oFhirModel.readLatestVersionOfResource(sPatientPath, fnSuccessCallback);
 	});
 
-});
\ No newline at end of file
+	/***
+	 * The request handle gets deleted only when the request is completed
```

Comment:
The request handle has to be deleted in case of success AND failure before the jQuery.complete hook is executed. In case of an update call with version read (direct request) the request handle has to be deleted right in the beginning of the success or failure to trigger the update call.

Reply:
I would use /** TEXT */. Not jsdoc syntax here

Reply:
done